source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,225 @@ Hardware Checklist
+Record the results for later reference.
+
+#. Test the drive's performance during write opeartions
+
+   This tests checks a drive's ability to write new data (uncached) to disk by creating a specified number of blocks at up to a certain number of bytes at a time to mimic how a drive would function with writing uncached data. 
+   
+   .. code-block::
+      :class: copyable
+
+      dd if=/dev/zero of=/mnt/driveN/testfile bs=128k count=80000 oflag=direct conv=fdatasync > dd-write-drive1.txt
+
+   Replace ``driveN`` with the path for the disk you are testing.
+
+   .. list-table::
+      :widths: auto
+      :width: 100%
+
+      * - ``dd``
+        - The command to copy and paste data.
+      * - ``if=/dev/zero``
+        - Read from ``/dev/zero``, an system-generated endless stream of 0 bytes used to create a file of a specified size
+      * - ``of=/mnt/driveN/testfile``
+        - Write to ``/mnt/driveN/testfile``
+      * - ``bs=128k``
+        - Write up to 128,000 bytes at a time
+      * - ``count=80000``
+        - Write up to 80000 blocks of data
+      * - ``oflag=direct``
+        - Use direct I/O to write to avoid data from caching
+      * - ``conv=fdatasync``
+        - Physically write output file data before finishing
+      * - ``> dd-write-drive1.txt``
+        - Write the contents of the operation's output to ``dd-write-drive1.txt`` in the current working directory
+
+   The operation returns the number of files written, total size written in bytes, the total length of time for the operation (in seconds), and the speed of the writing in some order of bytes per second.
+
+#. Test the drive's performance during read operations
+
+   .. code-block::
+      :class: copyable
+
+      dd if=/mnt/driveN/testfile of=/dev/null bs=128k iflag=direct > dd-read-drive1.txt
+
+   Replace ``driveN`` with the path for the disk you are testing.
+
+   .. list-table::
+      :widths: auto
+      :width: 100%
+
+      * - ``dd``
+        - The command to copy and paste data
+      * - ``if=/mnt/driveN/testfile``
+        - Read from ``/mnt/driveN/testfile``; replace with the path to the file to use for testing the drive's read performance
+      * - ``of=/dev/null``
+        - Write to ``/dev/null``, a virtual file that does not persist after the operation completes
+      * - ``bs=128k``
+        - Write up to 128,000 bytes at a time
+      * - ``count=80000``
+        - Write up to 80000 blocks of data
+      * - ``iflag=direct``
+        - Use direct I/O to read and avoid data from caching
+      * - ``> dd-read-drive1.txt``
+        - Write the contents of the operation's output to ``dd-read-drive1.txt`` in the current working directory
+
+   Use a sufficiently sized file that mimics the primary use case for your deployment to get accurate read test results.
+   
+   Small files have a size < 1MB.
+   Large files are from 1MB — 1GB.
+   Really large files are > 1GB.
```

Comment:
```suggestion
The following guidelines may help during performance testing:

- Small Files : < 128KB
- Normal Files: <1GB
- Large Files: > 1GB
```

I realized in re-reading that our small file logic specifically kicks in at 128KB so that's probably where we want to focus testing.

Reply:
👍